internal/credsretriever/refreshing_cache_test.go

```diff
@@ -450,20 +450,6 @@ func TestCachedCredentialRetriever_GetIamCredentials_ActiveRequestCaching(t *tes
 				sampleResponseOne,
 			},
 		},
-		{
-			name: "calls with errors",
-			requests: []credentials.EksCredentialsRequest{
-				sampleRequestOne,
-			},
```

Comment:
We should also add new tests to explicitly verify that the error is not cached

Reply:
> We should also add new tests to explicitly verify that the error is not cached

That in memory cache is explicitly removed